tests/users.spec.ts

```diff
@@ -22,13 +22,28 @@ describe('users', () => {
     await userRepository.createMany([user1, user2]);
   });
   describe('/users GET', () => {
-    it('should return all users', (done: jest.DoneCallback) => {
+    it('should return all users', async (done: jest.DoneCallback) => {
+      const user: User = await factory.attrs('User');
+      const noEncryptPass = user.password;
+      user.password = await encrypt(user.password);
+      await userRepository.createMany([user]);
+      // const listUsers: User[] = await factory.createMany('User', 8);
+      // await userRepository.createMany(listUsers);
+
       request(app)
-        .get('/users')
+        .post('/users/sessions')
+        .send({ email: user.email, password: noEncryptPass })
         .expect(200)
-        .then((res: request.Response) => {
-          expect(res.body.length).toBe(2);
-          done();
+        .then((res1: request.Response) => {
+          request(app)
+            .get('/users')
+            .set({ Authorization: res1.body.token })
```

Comment:
you can create a valid token in the code.
no need to call token endpoint.

Reply:
created token in the code